src/renderer/runner.ts

```diff
@@ -362,12 +369,15 @@ export class Runner {
    * or the user selected electron version
    */
   private async runFiddle(params: RunFiddleParams): Promise<RunResult> {
-    const { version, dir } = params;
+    const { version, dir, runFromAsar } = params;
     const { pushOutput, flushOutput, executionFlags } = this.appState;
     const env = this.buildChildEnvVars();
 
     // Add user-specified cli flags if any have been set.
     const options = [dir, '--inspect'].concat(executionFlags);
+    if (runFromAsar) {
+      options.push('runFromAsar');
+    }
```

Comment:
This should be added to `StartFiddleParams`, and then it will be handled automatically by the `...params` in the call to `window.ElectronFiddle.startFiddle` below. This will also require changes in `src/main/fiddle-core.ts` to plumb it into the call to `runner.spawn`.